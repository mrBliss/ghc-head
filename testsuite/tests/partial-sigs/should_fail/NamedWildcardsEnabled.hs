{-# LANGUAGE NamedWildcards #-}
module NamedWildcardsEnabled where

foo :: _a -> _b
foo x = not x

