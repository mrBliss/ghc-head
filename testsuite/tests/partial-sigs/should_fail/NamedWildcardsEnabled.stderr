
NamedWildcardsEnabled.hs:4:8:
    Instantiated wildcard ‘_a’ to: Bool
      in the type signature for foo :: _a -> _b
      at NamedWildcardsEnabled.hs:4:8-15
    The complete inferred type is: foo :: Bool -> Bool
    To use the inferred type, enable PartialTypeSignatures

NamedWildcardsEnabled.hs:4:14:
    Instantiated wildcard ‘_b’ to: Bool
      in the type signature for foo :: _a -> _b
      at NamedWildcardsEnabled.hs:4:8-15
    The complete inferred type is: foo :: Bool -> Bool
    To use the inferred type, enable PartialTypeSignatures
